language/de/ucp.php

```diff
@@ -447,20 +447,22 @@
 	'NO_OLDER_PM'				=> 'Keine älteren Nachrichten.',
 	'NO_PASSWORD_SUPPLIED'		=> 'Du kannst dich nicht ohne Passwort anmelden.',
 	'NO_RECIPIENT'				=> 'Kein Empfänger angegeben.',
+	'NO_RESET_TOKEN'			=> 'Kein Token zum Zurücksetzen des Passworts angegeben.',
```

Comment:
Ich würde hier "Es wurde kein Token zum Zurücksetzen des Passworts angegeben" nehmen, da die englische Fassung auch etwas ausführlicher ist (You did not...)